pkg/parse/run.go

```diff
@@ -130,7 +130,6 @@ func Run(ctx context.Context, p Parser) {
 			}
 			run(ctx, p, trigger, state)
 
-			retryTimer.Reset(opts.retryPeriod)               // Schedule retry attempt
```

Comment:
How does this change the interaction with the `readyToRetry` logic? It seems like that's independent from this timer

Reply:
You are right. I just tested our main branch, it does still support the smart retry. 

Let me go back to Morten's complaint to see whether I missed anything. Thanks!